client/my-sites/stats/stats-purchase/stats-purchase-personal.tsx

```diff
@@ -11,22 +13,27 @@ interface PersonalPurchaseProps {
 }
 
 const AVERAGE_PRICE_INFO = '$6';
```

Comment:
Could we note that we'll want to pull this information from the API in a future PR? We don't have immediate plans for creating this endpoint, so we may also need the ability to hide the text snippet associated with this figure.